simp.json.template

```diff
@@ -1,42 +1,56 @@
 {
-  "__": " SIMP Packer manifest to run tests for simp-core or any module     ",
-  "__": "  that requires a fully working puppet server.                     ",
-  "__": "                                                                   ",
-  "__": " USAGE:                                                            ",
+  "__": " SIMP Packer manifest to run tests for simp-core or any module       ",
+  "__": "  that requires a fully working puppet server.                       ",
+  "__": "                                                                     ",
+  "__": " USAGE:                                                              ",
   "__": "   `packer build -var-file=vars.json simp.json`              ",
-  "__": "                                                                   ",
-  "__": " NOTES:                                                            ",
-  "__": "   - The first `<wait10>`s are killing time until SSH can connect  ",
-  "__": "   - The comments starting with ## below will be parsed out by the ",
-  "__": "     start up script.                                              ",
-  "__": "   - You should not need change this file.  SIMP_PACKER environment",
-  "__": "     variables and the simp_conf.yaml file passed should be used   ",
-  "__": "     to changed behavior.                                          ",
-  "variables" :
-    {
-      "vagrant_password": "vagrant",
-      "simp_conf_file": "/var/local/simp/files/simp_conf.yaml",
-      "ruby_path": "/opt/puppetlabs/puppet/bin/ruby"
-    },
+  "__": "                                                                     ",
+  "__": " NOTES:                                                              ",
+  "__": "   - You should not need change this file.                           ",
+  "__": "   - Change behaviors using:                                         ",
+  "__": "     * SIMP_PACKER environment variables                             ",
+  "__": "     * the `simp_conf.yaml` file                                     ",
+  "__": "   - The first `<wait10>`s are killing time until SSH can connect    ",
+  "__": "   - The comments starting below (`//` or `#`) will be parsed out by ",
+  "__": "     the start-up script.                                            ",
+  "__": "                                                                     ",
+  //
+  // Template User variables:
+  //
+  //   https://www.packer.io/docs/templates/user-variables.html
+  //
+  "variables" : {
+    "vagrant_password": "vagrant",
+    "simp_conf_file": "/var/local/simp/files/simp_conf.yaml",
+    "ruby_path": "/opt/puppetlabs/puppet/bin/ruby",
+    // "ssh_private_key_file": "/home/ctessmer/.ssh/vagrant.key",
+    "ssh_agent_auth": "true",
```

Comment:
are these new variables documented anywhere?

Reply:
@lnemsick-simp No—in fact, _none_ of the user variables are documented yet.